config/routes.rb

```diff
@@ -5,6 +5,10 @@
     get :approve
   end
 
+  resources :partner_requests, only: [:new, :create] do
```

Comment:
only: [:new, :create, :show] and get rid of the thanks

Reply:
Should probably have an index page too now that I think about it. They'll probably want to view their old requests.